public/index.html

```diff
@@ -1,13 +1,17 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="" data-bs-theme="dark">
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lexend:wght@300;400;500;600;700&family=Lekton:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
```

Comment:
Load these files to `third-party`. Explorer should be able to launch in an air-tight environment without Internet access.